Blockzilla/OverlayView.swift

```diff
@@ -176,7 +176,23 @@ class OverlayView: UIView {
         let attributedString = NSMutableAttributedString(string: localizedStringFormat, attributes: [.foregroundColor: UIConstants.Photon.Grey10])
         let phraseString = NSAttributedString(string: phrase, attributes: [.font: UIConstants.fonts.copyButtonQuery,
                                                                            .foregroundColor: UIConstants.Photon.Grey10])
-
+		
+		if phrase != searchQuery {
+			let searchString = NSAttributedString(string: searchQuery, attributes: [.font: UIConstants.fonts.copyButtonQuery,
+																			   .foregroundColor: UIConstants.Photon.Grey10])
```

Comment:
Looks like something weird went on with the formatting here. Are you able to fix this and line 188 to match the rest of the formatting?